components/eam/src/physics/cam/ndrop.F90

```diff
@@ -58,6 +58,8 @@ module ndrop
                        (/ 0.02_r8, 0.05_r8, 0.1_r8, 0.2_r8, 0.5_r8, 1.0_r8 /)
 character(len=8) :: ccn_name(psat)= &
                     (/'CCN1','CCN2','CCN3','CCN4','CCN5','CCN6'/)
+character(len=8) :: ccnmair_name(psat)= &
```

Comment:
I think it would be a good idea to add a comment in the code explaining why these (largely redundant) additional CCN diagnostics are being added (i.e. for generating SPA inputs)